### PR TITLE
HLS S3 credentials

### PR DESCRIPTION
**Pull request by Karolk99**
https://github.com/jellyfish-dev/jellyfish/pull/117

### CODE REVIEW

File: docs/getting_started/components/hls.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ After a meeting that integrates an HLS component (with the persistent option set
 To manage this recording, use the [Recording API](../../for_developers/api_reference/rest_api#tag/recording).
 The recording is also available as HLS Video On Demand (VOD) [API](../../for_developers/api_reference/rest_api#tag/recording/operation/getRecordingContent).
 
+## Store HLS stream on S3
+You can directly store your HLS streams on S3. This solution will automatically send your streams to an AWS bucket right after the end of your meetings.
```
Comment (mickel8) [inline docs/getting_started/components/hls.md:41]: Please start every sentence from a new line
```suggestion
You can directly store your HLS streams on S3. 
This solution will automatically send your streams to an AWS bucket right after the end of your meeting.
```

File: docs/getting_started/components/hls.md [hunk 2]
```diff
@@ -37,6 +37,10 @@ After a meeting that integrates an HLS component (with the persistent option set
 To manage this recording, use the [Recording API](../../for_developers/api_reference/rest_api#tag/recording).
 The recording is also available as HLS Video On Demand (VOD) [API](../../for_developers/api_reference/rest_api#tag/recording/operation/getRecordingContent).
 
+## Store HLS stream on S3
```
Comment (mickel8) [inline docs/getting_started/components/hls.md:40]: We also need to add s3 configuration options in the section above

**Review by mickel8 — CHANGES_REQUESTED** [review]

File: docs/getting_started/components/hls.md (hunk 2 above)
Comment (Karolk99) [reply docs/getting_started/components/hls.md:40]: Currently, we don't support s3 streams in recordings so I don't see the point.

File: docs/getting_started/components/hls.md [hunk 3]
```diff
@@ -22,6 +22,8 @@ See [API](../../for_developers/api_reference/rest_api#tag/room/operation/create_
 * `persistent` (boolean, default: false) - whether the stream should be saved or not
 * `targetWindowDuration` (positive integer, default: null) - represents the duration, in seconds, of the live streaming content to be
     maintained in a rolling window. If set to null (default), the entire stream will be available.
+* `s3`` (object, default: nil) - AWS S3 credentials. If credentials are set, the stream will be saved to the specified bucket.
```
Comment (mickel8) [inline docs/getting_started/components/hls.md:25]:
```suggestion
* `s3` (object, default: nil) - AWS S3 credentials. If credentials are set, the stream will be saved to the specified bucket.
```

**Review by mickel8 — APPROVED** [review]

**Pull request merged**